src-tauri/src/operations.rs

```diff
@@ -148,6 +159,47 @@ impl ProjectManager for LocalProjectManager {
         Ok(project.clone())
     }
 
+    // Load project config
+    fn load_project_config(&self, project: Project) -> io::Result<ProjectConfig> {
+        let project_config_path = self
+            .projects_dir()
+            .join(project.name)
+            .join(PROJECT_CONFIG_FILE);
+        if !project_config_path.exists() {
+            return Err(io::Error::new(
+                io::ErrorKind::NotFound,
+                "project config not found",
+            ));
+        }
+
+        let file = fs::File::open(&project_config_path)?;
+        let project_config = serde_json::from_reader(file)?;
+        Ok(project_config)
+    }
+
+    // Save project config
+    fn save_project_config(
+        &self,
+        project: Project,
+        project_config: ProjectConfig,
+    ) -> io::Result<()> {
```

Comment:
I was thinking about was letting the user specify the cloud project id during the project creation, at which point we won't have `project` yet. On the other hand, we can probably invoke this command from `create_project` and pass the newly created project, right?

Reply:
correct, you can call it from there.

Although, I would consider the `project_id` specification optional if presented on project creation. (I might never use the cloud for now)

Reply:
Yes, it should definitely be optional 👍